guides/v2.1/config-guide/cli/config-cli-subcommands-i18n.md

```diff
@@ -158,7 +170,7 @@ If necessary, you can specify several parent packages. The parent packages are a
 
 #### Example of language inheritance {#m2devgde-xlate-inheritancework}
 
-Suppose a language package descends from two other packages, and that those packages also have parent and "grandparent" packages.
+It is possible a language package descends from two other packages, and that those packages also have parent and "grandparent" packages.
```

Comment:
I like "suppose" better. "It is possible" sounds like you're describing a very technical fine point.  If you want to change the suppose, then say "In this example," or something similar.